packages/components/src/focal-point-picker/stories/index.tsx

```diff
@@ -0,0 +1,89 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+/**
+ * Internal dependencies
+ */
+import FocalPointPicker from '..';
+
+const meta: ComponentMeta< typeof FocalPointPicker > = {
+	title: 'Components/FocalPointPicker',
+	component: FocalPointPicker,
+	parameters: {
```

Comment:
Let's force the `label` control to be just text, currently it's an object

![Screenshot 2022-09-07 at 10 01 25](https://user-images.githubusercontent.com/1083581/188824049-8144ce89-9d56-4b6c-a059-ae632e57c86d.png)